lua/wikis/splatoon/MatchSummary.lua

```diff
@@ -5,120 +5,108 @@
 -- Please see https://github.com/Liquipedia/Lua-Modules to contribute
 --
 
-local CustomMatchSummary = {}
-
 local Lua = require('Module:Lua')
 
 local Array = Lua.import('Module:Array')
-local MapTypeIcon = Lua.import('Module:MapType')
+local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
 local Operator = Lua.import('Module:Operator')
-local String = Lua.import('Module:StringUtils')
-local Table = Lua.import('Module:Table')
 local WeaponIcon = Lua.import('Module:WeaponIcon')
 
-local DisplayHelper = Lua.import('Module:MatchGroup/Display/Helper')
-local MatchSummary = Lua.import('Module:MatchSummary/Base')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
 local MatchSummaryWidgets = Lua.import('Module:Widget/Match/Summary/All')
+local MatchSummary = Lua.import('Module:MatchSummary/Base')
 local WidgetUtil = Lua.import('Module:Widget/Util')
 
-local NON_BREAKING_SPACE = '&nbsp;'
+
+local CustomMatchSummary = {}
 
 ---@param args table
 ---@return Html
 function CustomMatchSummary.getByMatchId(args)
-	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {width = '490px', teamStyle = 'bracket'})
+	return MatchSummary.defaultGetByMatchId(CustomMatchSummary, args, {width = '500px', teamStyle = 'bracket'})
+end
+
+---@param match MatchGroupUtilMatch
+---@return Widget[]
+function CustomMatchSummary.createBody(match)
+	return WidgetUtil.collect(
+		Array.map(match.games, function(game, gameIndex)
+			return CustomMatchSummary._createGameRow(game, gameIndex, match.game)
+		end)
+	)
 end
 
----@param date string
 ---@param game MatchGroupUtilGame
 ---@param gameIndex integer
+---@param gameTitle string
 ---@return Widget?
-function CustomMatchSummary.createGame(date, game, gameIndex)
-	local weaponsData = Array.map(game.opponents, function(opponent)
-		return Array.map(opponent.players, Operator.property('weapon'))
-	end)
+function CustomMatchSummary._createGameRow(game, gameIndex, gameTitle)
+	if not game.map then
+		return
+	end
+
+	local function makeTeamSection(opponentIndex)
+		local opponent = game.opponents[opponentIndex] or {}
+		local weaponsData = Array.map(opponent.players or {}, Operator.property('weapon'))
+
+		local score = opponent.score
+		if score and game.mode == 'turf war' then
+			score = score .. '%'
+		end
+		local scoreDisplay = DisplayHelper.MapScore({score = score}, game.status)
+
+		return {
+			CustomMatchSummary._createWeaponsDisplay{
+				data = weaponsData,
+				flip = (opponentIndex == 2),
+				game = gameTitle
```

Comment:
why do we need match.game
wouldn't game.game suffice?
would allow simplifying quite a bit